lisa/features/disks.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import List
+
+from lisa.feature import Feature
+from lisa.util import SwitchableMixin
+
+
+class DiskEphemeral(Feature, SwitchableMixin):
+    @classmethod
+    def name(cls) -> str:
+        return "DiskEphemeral"
+
+    @classmethod
+    def can_disable(cls) -> bool:
+        return True
+
+    def enabled(self) -> bool:
+        return True
+
+    @staticmethod
+    def get_disk_id() -> str:
+        raise NotImplementedError
+
+
+class DiskPremiumLRS(Feature, SwitchableMixin):
+    @classmethod
+    def name(cls) -> str:
+        return "DiskPremiumLRS"
```

Comment:
It can be "cls.__name__". It avoids typo and if the class name is changed, it still works.